includes/Core/Authentication/Authentication.php

```diff
@@ -297,7 +297,7 @@ public function get_connect_url() {
 			'splash',
 			array(
 				'googlesitekit_connect' => 1,
-				'nonce'                 => wp_create_nonce( 'connect' ),
+				'nonce'                 => wp_generate_password(),
```

Comment:
This needs to remain as a WP nonce or it will fail verification
https://github.com/google/site-kit-wp/blob/35eef8d555bd8abe887244ae7689385dbd3ad48c/includes/Core/Authentication/Authentication.php#L382-L386